### ISSUE TITLE
Only rejected shares on btcz.suprnova.cc depending on order of parameters

### ISSUE THREAD

**Issue by s0ftcorn** [funakoshi2718/funakoshi-miner#29]
```
funakoshiMiner -cd 0 -l btcz.suprnova.cc:6588 -u userName.workerName -p x -temp-max 80 -144
```
 gives only rejected shares.

while 
```
funakoshiMiner -cd 0 -l btcz.suprnova.cc:6588 -u userName.workerName -p x -144 -temp-max 80
```
works perfectly fine.

**Comment by funakoshi2718** [comment]
-temp-max requires 2 additional arguments: higher bound and lower bound.
Thus the -144 is treated as the second augument of temp-max.

Example:    -temp-max 80 70   (tells the miner to pause mining when temperature reaches 80 and
to resume when temparature drops to 70).

**Comment by funakoshi2718** [comment]
This bug has been fixed in release v3.2
Thank you very much for reporting the bug.
